packages/dso-toolkit/components/Componenten/label-css/label-css.njk

```diff
@@ -0,0 +1,11 @@
+<span {{ className('dso-label', [status, 'dso-label-' + status], [small, 'dso-label-small'], [hovered and removable, 'dso-label-hover']) }}>
+  {% if status %}
+    <span class="sr-only">{{ status }}: </span>
+  {% endif %}
+  {{ label }}
+  {% if removable %}
+    <button type="button">
+      <dso-icon icon="times"></dso-icon>
```

Comment:
In een CSS component gebruik maken van een Web Component?

Reply:
Woops, is nu aangepast